### ISSUE TITLE
In line ammeter does not read out value

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/circuit-construction-kit-common#1268]
**Device**
Samsung
**OS**
Win 11
**Browser**
Chrome + Jaws
**Problem Description**
For https://github.com/phetsims/qa/issues/1360
The contact ammeter will read out its value when you focus on it, but the in line ammeter does not. I cannot seem to find any way for the description to say what the in line ammeter is reading. Also seen in a11y view.
